### ISSUE TITLE
Stage Channel

### ISSUE THREAD

**Issue by AayanAsim746** [galnir/Master-Bot#491]
**Explain your suggestion**
So there is a new update on discord the stage channel so I was hoping the bot could join that channel

**Comment by galnir** [comment]
I'll look into it

**Comment by Natemo6348** [comment]
discord.js will need to be updated to v12.5.3 as well btw

**Comment by galnir** [comment]
Stage channels are not supported by discord.js at the moment

**Comment by Natemo6348** [comment]
Yes, but I remember seeing that without that update, stage channels could hard crash the bot, I will look for exactly what they said

**Comment by Natemo6348** [comment]
Couldn't find exactly what they said, but I do know that previous versions are deprecated because of issue with stage channels

**Comment by galnir** [comment]
> Couldn't find exactly what they said, but I do know that previous versions are deprecated because of issue with stage channels

v11 is deprecated. This bot uses v12 and they(discord.js) didn't finish working on supporting stage channels. Stage channels are not even documented in Discord's official API, atleast from what I've seen

**Comment by LilDerp-IsBetter** [comment]
@galnir Ive checked the repo of discord.js and have seen that they have added stage channels now. Are you able to add this now?

**Comment by Bacon-Fixation** [comment]
It was added to the GitHub repo, but still no Documentation, and hasn't been added to the "Stable" release yet
kinda need to wait till discordjs fully supports it and documented for me to work in it at this time.

Much Love
-Bacon

**Comment by Bacon-Fixation** [comment]
seems that Stage Channels may be part of the [discord.js v13](https://github.com/monbrey/guide/blob/master/guide/additional-info/changes-in-v13.md) update along with slash commands,
still no announcement date that i can find
but i have been playing around with the github version 
so far i can get the bot to join a stage channel with v13.0.1(beta), 
it has been a slow learning experience as there is little to no info for this yet

i haven't gotten it fully converted to v13 (pagination embeds needs an update)

just thought i would give a heads-up since its been over a month
![image](https://user-images.githubusercontent.com/12632936/120119059-3d538300-c15b-11eb-8ec0-5f16d8f1f220.png)



Much Love
-Bacon

**Comment by Bacon-Fixation** [comment]
I have it working, but it's not a normal install procedure (until they release it) 
but i really wanted a head start :D [Master-bot running discord.js v13.0.1](https://github.com/Bacon-Fixation/Master-Bot/tree/Discordjs-v12-to-v13-update) 
to get running `npm install` first then need to delete the existing `./node_modules/discord.js` folder and rename `./node_modules/discord.js-v13` to `discord.js` 

reason for the deletion and rename of folders was to have the pagination Embed module to install correctly <3

Much Love
-Bacon

**Comment by Natemo6348** [comment]
well, I just found out that commando is now deprecated because discord.js v13 does not have structures, what could this mean for the future of Master-Bot?

**Comment by galnir** [comment]
> well, I just found out that commando is now deprecated because discord.js v13 does not have structures, what could this mean for the future of Master-Bot?

I'm rewriting the bot without a framework, should be ready soon

**Comment by Natemo6348** [comment]
> > well, I just found out that commando is now deprecated because discord.js v13 does not have structures, what could this mean for the future of Master-Bot?
> 
> I'm rewriting the bot without a framework, should be ready soon

oh ok, nice, cant wait!
